jakexks/go.d.plugin
# Renaming and refactoring

Do not be offended by the boss crufting up all your shit. I am just a massive cruftlord!

Unfortunately when you rename a file go decides to cruft it up.

Changes:

- Rename collector to Micro
- Update variable names
- Use latest go micro since debug proto has moved
- Scrape service entries from registry on a 10 second basis
- Pull entire service from registry in case data is not there in List
- Check "protocol" in metadata so we only look at "mucp" for now
- Strip unused timeouts
- Fix the breaking tests
